src/og/control/Control.ts

```diff
@@ -1,17 +1,22 @@
-"use strict";
+import { Planet, Renderer } from "../index";
 
 /**
  * Base control class for implementing renderer controls.
  * All other controls extend from this class.
- * @class
- * @param {Object} [options] - Control activation options:
- * @param {Boolean} [options.autoActivated=true] - If true - calls initialize function after the renderer assigning.
  */
-class Control {
+export class Control {
+    _id = 0;
+    _name: string;
+    planet: Planet | null;
+    _initialized: boolean;
+    renderer: any;
+    autoActivate: any;
+    _active: boolean;
+    static __counter: any;
```

Comment:
Let's use proper TS static properties for __counter and follow Control._staticCounter . I use:
```
static __counter__: number = 0;

constructor(){
    this._id = Control.__counter__++;
}
```

now we do not need static getter/setter methods like __staticCounter, it should be removed.